zeebe/engine/src/main/java/io/camunda/zeebe/engine/processing/job/behaviour/JobUpdateBehaviour.java

```diff
@@ -54,28 +50,27 @@ public Either<String, JobRecord> getJob(final long jobKey, final TypedRecord<Job
   }
 
   public Optional<String> updateJobRetries(
-      final long jobKey, final JobRecord jobRecord, final TypedRecord<JobRecord> command) {
-    final int retries = command.getValue().getRetries();
+      final long jobKey,
+      final int retries,
+      final JobRecord jobRecord,
+      final TypedRecord<JobRecord> command) {
```

Comment:
🔧 I think we don't need the `command` anymore, do we? The retries are already provided.